packages/dso-toolkit/components/Voorbeelden/Homepage.njk

```diff
@@ -1,83 +1,211 @@
+
+<style>
+  .dso-tile-grid {
+      grid-template-columns: repeat(3, 1fr);
+  }
+  
+  @media only screen and (max-width: 480px) {
+ .dso-tile-grid {
+      grid-template-columns: repeat(2, 1fr);
+  }
+}
+</style>
 <header>
   <div class="dso-logo-bar">
-    <img src="../../images/omgevingsloket.png" alt="Omgevingsloket" class="logo" />
+    <img style="margin-top: 8px" src="../../logos/omgevingsloket.svg" alt="Omgevingsloket" class="logo"/>
```

Comment:
De styling in het `style` attribuut moet weg ergens anders worden ondergebracht. Waarom zet je dit niet in `_header.scss` in een selector `header .logo` ?

Reply:
Verplaatst naar header.logo